cli/src/main/kotlin/commands/DownloaderCommand.kt

```diff
@@ -218,8 +241,27 @@ class DownloaderCommand : CliktCommand(name = "download", help = "Fetch source c
                             allowMovingRevisions
                         )
 
-                        if (archiveMode == ArchiveMode.ENTITY && archive(pkg, dir)) {
-                            dir.safeDeleteRecursively(baseDirectory = outputDir)
+                        val pkgCategories = getCategoriesForPackage(
```

Comment:
Maybe `licenseCategories` instead of `pkgCategories`.

Reply:
My intention was to specifically mention `pkg` because these are the license categories for the package.
I changed it to `packageLicenseCategories`.